.github/workflows/callable-partner-driver.yml

```diff
@@ -0,0 +1,140 @@
+name: Test Partner Driver
+
+on:
+  workflow_call:
+    inputs:
+      DRIVER:
+        description: 'Driver to use'
+        type: string
+        required: true
+      MB_REF:
+        type: string
+        required: true
+      PARTNER_REF:
+        type: string
+        required: true
+      REPO:
+        type: string
+        required: true
+      DOCKER:
+        type: boolean
+        required: true
+    secrets:
+      OP_SERVICE_ACCOUNT_TOKEN:
+        required: true
+
+jobs:
+  run-tests:
+    timeout-minutes: 45
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./metabase
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: registry
+
+      - name: Checkout Metabase Repo
+        uses: actions/checkout@v4
+        with:
+          path: metabase
+          repository: metabase/metabase
+          ref: ${{ inputs.MB_REF }}
+
+      - name: Checkout Driver Repo
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ inputs.REPO }}
+          path: metabase/modules/drivers/${{ inputs.DRIVER }}
+          ref: ${{ inputs.PARTNER_REF }}
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 21
```

Comment:
we might want to test this on a few different versions: at least 11 and 21

Reply:
I think 21 should be sufficient for now, as we get more in the weeds having a mechanism for multiple java versions would make sense.